__playwright-tests__/fixtures/dynamic-content.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Dynamic Content E2E Tests</title>
+        <style>
+            * {
+                color: white;
+            }
+            body {
+                background-color: teal;
+            }
+
+            #time-e2e-test-ignore {
+                font-size: 16pt;
+                width: 100vw;
+            }
+
+        </style>
+    </head>
+    <body>
+        <div class="container" style="background-color: teal">
+            <h1>Hello There! The current time is:</h1>
+            <div data-chromatic="ignore" class="chromatic-ignore" id="time-e2e-test-ignore"></div>
```

Comment:
Can we test the `data-chromatic` attribute and the `chromatic-ignore` class on separate elements to ensure they both work?